controllers/common/common_status_update.go

```diff
@@ -68,9 +69,10 @@ func RootStatusUpdate(ctx context.Context, c client.Client, rootPolicy *policies
 // a required lookup fails.
 func GetPolicyPlacementDecisions(ctx context.Context, c client.Client,
 	instance *policiesv1.Policy, pb *policiesv1.PlacementBinding,
-) (decisions []appsv1.PlacementDecision, placements []*policiesv1.Placement, err error) {
+) (decisions clusterv1alpha1.RolloutResult, placements []*policiesv1.Placement, err error) {
```

Comment:
Does the decisions replace by RolloutResult?

Reply:
Oops, good catch!